templates/apps/bug-report.html

```diff
@@ -24,21 +24,21 @@
 </div>
 
 <div class="form-group">
-    <label>Contact Info</label>
-    <input type="text" placeholder="Discord Tag or Email." value="{{name}}"class="issuer">
+    <label>{{localize "Contact Info"}}</label>
+    <input type="text" placeholder="{{localize 'Discord Tag or Email'}}." value="{{name}}"class="issuer">
```

Comment:
This does not have a key and is preferably an actual key, like `BUGREPORT.ContactPlaceholder`